packages/esm-patient-vitals-app/src/vitals-biometrics-form/vitals-biometrics-form.workspace.tsx

```diff
@@ -588,7 +588,7 @@ const VitalsAndBiometricsForm: React.FC<VitalsAndBiometricsFormProps> = ({
             className={styles.errorNotification}
             lowContrast={false}
             onClose={() => setHasInvalidVitals(false)}
-            title={t('vitalsAndBiometricsSaveError', 'Error saving vitals and biometrics')}
+            title={t('vitalsAndBiometricsSaveError', 'Error saving Vitals and Biometrics')}
```

Comment:
This fixes an i18next-parser warning about duplicate values for this key by reusing the existing key/value pair.